src/pages/trade-types/multiplier/index.tsx

```diff
@@ -14,9 +14,9 @@ const MarketsAvailable = Loadable(() => import('./_markets-available'))
 const WhatAreMultiplier = Loadable(() => import('./_what-are-options'))
 
 const meta_attributes = {
-    og_title: localize('Multipliers trading | Trade multipliers on Deriv'),
+    og_title: localize('_t_Multipliers trading | Trade multipliers on Deriv_t_'),
     og_description: localize(
-        'Explore trading on markets with multipliers on Deriv. Trade forex, cryptocurrencies, and more and maximise potential profit without risking more than your stake.',
+        '_t_Explore trading on markets with multipliers on Deriv. Trade forex, cryptocurrencies, and more and maximise potential profit without risking more than your stake._t_',
     ),
 }
```

Comment:
Let's pass attributes inside the `SEO` component

![Screenshot 2023-03-06 at 10 25 39](https://user-images.githubusercontent.com/103182473/223044934-3bc45f4c-a960-4644-b48e-17801730617b.png)